modules/rampart-trust/pom.xml

```diff
@@ -89,6 +89,14 @@
             <groupId>xerces.wso2</groupId>
             <artifactId>xercesImpl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.framework</groupId>
+            <artifactId>org.wso2.carbon.identity.application.mgt</artifactId>
+        </dependency>
```

Comment:
@Rod-yan the build seems to fail due to a compilation error. Seems the new framework dependencies you have introduced are not in the parent pom. Can you please verify it locally?